File: Pseudo2.java
// Java Pseudocode 2

public class Pseudo2 {
    public static int CalculateSum(int a, int b){
        return a+b;
    }
    public static void main(String[] args) {

    int i,j,sum;
    i = 15;
    j = 25;

    if (i>j){
        System.out.println("i is greater than j");
    } else{
        sum = CalculateSum(i,j);
        System.out.println("Sum of i and j is: " + sum);
    }
    for (int h = 1; h <= 5; h++ ){
        System.out.println("Value of h is: " + h);
    }
    }
}

File: Pseudo2.cpp
// CPP Pseudocode 2

#include <iostream>
using namespace std;

int CalculateSum(int a,int b){
    return a+b;
};

int main(){

    int i,j,sum;
    i = 15;
    j = 25;

    if (i > j){
        cout << "i is greater than j" << endl;
    } else{
        sum = CalculateSum(i,j);
        cout << "Sum of i and j is: " << sum << endl;
    }

    for (int h = 1; h <= 5; h++){
        cout << endl;
        cout << "The value of h is: " << h << endl;
    }
}